cpp/src/arrow/ipc/read_write_test.cc

```diff
@@ -1288,7 +1288,8 @@ TYPED_TEST_P(TestSparseTensorRoundTrip, WithSparseCOOIndexRowMajor) {
   std::shared_ptr<SparseCOOIndex> si;
   ASSERT_OK(SparseCOOIndex::Make(TypeTraits<IndexValueType>::type_singleton(), {12, 3},
                                  {sizeof_index_value * 3, sizeof_index_value},
-                                 Buffer::Wrap(coords_values), &si));
+                                 Buffer::Wrap(coords_values))
+                .Value(&si));
```

Comment:
This works but could be more idiomatically written:
```c++
ASSERT_OK_AND_ASSIGN(si, SparseCOOIndex::Make(...));
```

Reply:
I got it.